src-tauri/info.plist

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSCameraUsageDescription</key>
+	<string>Request camera access</string>
+	<key>NSMicrophoneUsageDescription</key>
+	<string>Request microphone access</string>
```

Comment:
This goes on a different PR but on MacOS we can use `LSMinimumSystemVersion` to 13.4